modules/infinite_fia/infinite_fia.module

```diff
@@ -81,11 +81,11 @@ function infinite_fia_node_view(array &$build, EntityInterface $entity, EntityVi
         GA_CD_FIA_LOGGED_IN => $datalayer_variables['environment']['isLoggedIn'] ?: '',
         GA_CD_FIA_ARTICLE_NAME => $datalayer_variables['page']['name'] ?: '',
         GA_CD_FIA_CONTENT_TYPE => $datalayer_variables['page']['contentType'] ?: '',
-        GA_CD_FIA_CONTENT_SUB_TYPE => $datalayer_variables['page']['contentSubType'] ?: '',
+        GA_CD_FIA_CONTENT_SUB_TYPE => $datalayer_variables['page']['contentSubType'] ?? '',
         GA_CD_FIA_ARTICLE_PUBLISH_DATE => $datalayer_variables['page']['articlePublishDate'] ?: '',
         GA_CD_FIA_AUTHOR_NAME => $datalayer_variables['page']['authorName'] ?: '',
         GA_CD_FIA_CHANNEL_CHANNEL => $datalayer_variables['page']['category'] ?: '',
-        GA_CD_FIA_SUB_CATEGORY => $datalayer_variables['page']['subCategory'] ?: '',
+        GA_CD_FIA_SUB_CATEGORY => $datalayer_variables['page']['subCategory'] ?? '',
```

Comment:
what could possibly the reason, that the subCategory is not set? Do other problems result from that?

Reply:
I've discussed this with Nico and then talked to Sara. There are no known problems, Sara will open a ticket when she discovers one.